llvm/test/tools/sycl-post-link/spec-constants/multiple-composite-usages.ll

```diff
@@ -6,11 +6,11 @@
 ; once
 ;
 ; CHECK-LABEL: @foo1
-; CHECK: call %struct._ZTS3POD.POD @"_Z29__spirv_SpecConstantCompositeAstruct._ZTS1A.Aclass._ZTSN2cl4sycl3vecIiLi2EEE.cl::sycl::vec"({{.*}})
+; CHECK: call %struct._ZTS3POD.POD @"_Z29__spirv_SpecConstantCompositeA2_struct._ZTS1A.Aclass._ZTSN2cl4sycl3vecIiLi2EEE.cl::sycl::vec_Rstruct._ZTS3POD.POD"({{.*}})
```

Comment:
nit: couple interesting test cases:
- nested arrays (like `[2 x [3 x [4 x i16]]]`)
- make sure arrays of different length by same type result in different mangled name

Reply:
Added test for the second case in c88f6ffc433b324c3e3359d4887cc1c027ff2c64

Reply:
Added the first test case in bbcc8352a8571a17f2241025fa1bc8168e847813